Sources/FINNFilterImplementation/FINNFilterSelectionData.swift

```diff
@@ -0,0 +1,219 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import Foundation
+
+class FINNFilterSelectionData: NSObject {
+    private struct GeoKey {
+        static let latitude = "lat"
+        static let longitude = "lon"
+        static let radius = "radius"
+        static let locationName = "geoLocationName"
+    }
+
+    private(set) var selectionValues: [String: [String]]
+
+    init(selectionValues: [String: [String]]) {
+        self.selectionValues = selectionValues
+    }
+
+    convenience override init() {
+        self.init(selectionValues: [:])
+    }
+}
+
+extension FINNFilterSelectionData {
+    func setSelectionValues(_ values: [String], for key: String) {
```

Comment:
Do we have tests for methods in this class?

Reply:
Indirectly, there are tests for the public api implemented in ParameterBasedFilterInfoSelectionDataSource, I will have a look at if it makes sense to add test to this class too 🤔